CODEOWNERS

```diff
@@ -3,3 +3,15 @@
 
 # For anything not explicitly taken by someone else:
 * @open-telemetry/dotnet-contrib-approvers
```

Comment:
This top rule applies to anything not defined? Should we then just define AWS stuff below and let everything else fall to the default?

Reply:
lets come back to this. will try merging this pr now to unblock progress.